postman/collection.json

```diff
@@ -659,6 +677,57 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "Get the last active time-event.",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "d2008de1-89e8-485b-82d6-5ce1db5ffe7e",
+								"exec": [
+									"pm.test(\"Status code is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"",
+									"pm.test(\"Check the id of last active time event is correct\", function () {",
+									"    var jsonData = pm.response.json();",
+									"    pm.expect(jsonData.id === pm.globals.get(\"2-time-event-id\"));",
+									"});",
```

Comment:
I don't like this name `2-time-event-id`, What about `last-time-event-created-id`?
Clean Code practice are learned to be used :)

Reply:
Sorry for that. Those names were in the postman collection of the master branch, I only increased it one test, but I'm going to change it.